### ISSUE TITLE
Error in revdep-check with sjstats

### ISSUE THREAD

**Issue by strengejacke** [IndrajeetPatil/ggstatsplot#128]
I got following error during revdep-check, however, I'm not sure whether this is related to my package?

```
checking tests ... ERROR
  Running 'spelling.R'
  Running 'testthat.R' [120s]
Running the tests in 'tests/testthat.R' failed.
Last 13 lines of output:
  `using_function1` not equal to `results1`.
  target, current do not match when deparsed
  
  == testthat results  =========================================================================
  OK: 647 SKIPPED: 0 FAILED: 7
  1. Failure: lm_effsize_ci works (@test_lm_effsize_ci.R#68) 
  2. Failure: lm_effsize_ci works (@test_lm_effsize_ci.R#69) 
  3. Failure: lm_effsize_ci works (@test_lm_effsize_ci.R#74) 
  4. Failure: lm_effsize_ci works (@test_lm_effsize_ci.R#75) 
  5. Failure: parametric anova subtitles work (with NAs) (@test_subtitle_anova_parametric.R#141) 
  6. Failure: parametric anova subtitles with partial eta-squared and data with NAs (@test_subtitle_anova_parametric.R#367) 
  7. Failure: parametric t-test works (between-subjects without NAs) (@test_subtitle_t_parametric.R#63) 
  
  Error: testthat unit tests failed
  Execution halted
```

**Comment by IndrajeetPatil** [comment]
Yes, this is indeed related to `sjstats`. The failures 1 through 6 are due to `sjstats`. The estimates for eta- and omega-squared seem to have changed with the recent update.
I will modify my tests now to be compatible with `sjstats 0.17.3` and submit to `CRAN` once I see the newer version accepted. 

When do you plan to submit `sjstats 0.17.3` to `CRAN`? I'll be ready accordingly.

**Comment by strengejacke** [comment]
Hm, why did the estimates change? That shouldn't be, actually... Have you defined a tolerance? I'm planning to submit asap, so would you suggest that you submit first, or should I first submit sjstats?

**Comment by IndrajeetPatil** [comment]
I do define tolerance for the tests in question-
https://github.com/IndrajeetPatil/ggstatsplot/blob/c541f8dc005d4dda2373814a03e86d94a4bce612/tests/testthat/test_lm_effsize_ci.R#L67-L80

You can submit first and then I will submit. Please mention this interaction in `cran_comments.md` when you submit to CRAN, so that the CRAN people know that you have already informed me about the breaking changes and I've made the necessary changes in the development version of the package.

**Comment by strengejacke** [comment]
Ok, I think I found the issue. In previous sjstats-versions, `boot_ci()` (used when `partial = FALSE`) was fixed to ci.lvl 0.95, so in fact `ci.lvl` was ignored. Your test results above refer to ci.lvl 0.95, not 0.99. Now I have fixed this issue and you can change the ci.lvl in `boot_ci()`, which means that `eta_sq(partial = FALSE)` now returns the "correct" CIs for ci.lvl 0.99 - which of course breaks your tests.

**Comment by IndrajeetPatil** [comment]
Glad we got to the bottom of this! I will happily modify my tests now once the new version of `sjstats` is on `CRAN`.
